src/main/java/in/koreatech/koin/global/validation/UniqueIdValidator.java

```diff
@@ -0,0 +1,27 @@
+package in.koreatech.koin.global.validation;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+
+public class UniqueIdValidator implements ConstraintValidator<UniqueId, List<Integer>> {
+
+    @Override
+    public void initialize(UniqueId constraintAnnotation) {
+        ConstraintValidator.super.initialize(constraintAnnotation);
+    }
+
+    @Override
+    public boolean isValid(List<Integer> elements, ConstraintValidatorContext context) {
+        Set<Integer> set = new HashSet<>(elements.size());
+        for (Integer element : elements) {
+            if (!set.add(element)) {
+                return false;
+            }
+        }
+        return true;
+    }
```

Comment:
코드가 조금 클린하지 않다고 생각합니다. 람다식 도입은 어떻게 생각하시나요?
ex) 
```suggestion
    public boolean isValid(List<Integer> elements, ConstraintValidatorContext context) {
    return elements.stream().distinct().count() == elements.size();
}

```

Reply:
오 좋습니다 수정하도록하겠습니다